webpki-roots/Cargo.toml

```diff
@@ -1,28 +1,23 @@
 [package]
 name = "webpki-roots"
-version = { workspace = true }
+version = "0.26.0"
```

Comment:
These changes to the webpki-roots package metadata shouldn't be a part of this commit.

Reply:
I think now it is fixed as requested.